alacritty_terminal/src/term/mod.rs

```diff
@@ -587,16 +587,16 @@ impl<T> Term<T> {
         // Scroll selection.
         self.selection = self.selection.take().and_then(|s| s.rotate(self, &region, lines as i32));
 
+        // Scroll from origin to bottom less number of lines.
```

Comment:
You got any clue at all what this comment is supposed to tell me? It seems rather broken to me.

Let's maybe just change it up while we're at it. I think I might just suggest removing the comment completely since the line itself is pretty self-explanatory?

Reply:
> Let's maybe just change it up while we're at it.

Yes, I have done it without thinking...

> I think I might just suggest removing the comment completely since the line itself is pretty self-explanatory?

I agree too. Come to think of it, this comment may be incorrect. "from origin to `bottom`" but the function name is `scroll_up`!